src/ui/style/palette.scss

```diff
@@ -15,6 +15,7 @@ $secondary-disabled: #b2c3d1;
 
 // neutral (grey)
 $neutral: #babec2;
+$neutral-200: #E5EBF0;
```

Comment:
AFAI, the naming breaks the existing convention (`-variant-x`, `-dark/light`), and the `-200` suffix doesn't clearly indicate how this color differs from the base `$neutral`.
But I don't have a clear vision of the principles governing the palette.

Reply:
Yes, the color palettes in our design system have recently been updated, featuring clearer variations for each palette.

Here's an example of the new color palettes:
<img width="611" alt="image" src="https://github.com/okp4/dataverse-portal/assets/8344874/49d2e7a3-d3fd-4482-890f-b6ff75c079f6">

Refactoring the existing palettes seems essential for enhancing maintainability and readability. This can be tackled in a separate PR, once we have decided on an implementation strategy for the new palettes.

We can open a discussion to determine the most suitable solution for this matter.

As for the current PR, we can either leave it as is, provided we agree to take action on refactoring the palettes later, or directly apply the color in the specific context of the disabled button, which is currently its sole use.

Reply:
Okay, no worries. I'd suggest addressing the topic in a separate ticket (or github thread). We can consider leaving the current naming as it is. Thx.